modules/bitgo/example/js/btc/check-lightning-balance.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Check the lightning balance of a wallet.
+ *
+ * Copyright 2022 BitGo, Inc.  All Rights Reserved.
+ */
+const BitGoJS = require('bitgo');
+
+// change this to env: 'production' when you are ready for production
+const env = 'test';
+// change coin to 'btc' when working with production
+const coin = 'tbtc';
+
+const bitgo = new BitGoJS.BitGo({ env });
+
+// set your access token here
+const accessToken = '';
+
+// set your wallet from the YYYYY parameter here in the URL on app.bitgo-test.com
+// https://test.bitgo.com/enterprise/XXXXXXXXX/coin/tbtc/YYYYY/transactions
+const walletId = '';
+
+async function checkLightningBalance() {
+  bitgo.authenticateWithAccessToken({ accessToken });
+  const wallet = await bitgo.coin(coin).wallets().get({ id: walletId });
+  const lightning = wallet.lightning();
+
+  const lightningBalance = await lightning.getBalance();
+  console.log('Lightning Balance:', lightningBalance.balance);
+}
+
+checkLightningBalance().catch((e) => console.error(e));
```

Comment:
Nit but here (and anywhere else you're using `.catch` in these examples) you can just do
```suggestion
checkLightningBalance().catch(console.error);
```